markdownlint/docs/TOP008.md

```diff
@@ -20,4 +20,4 @@ You should use backticks instead
 
 Consistent formatting makes it easier to understand a document.
 
-Markdown lint's [MD048 rule](https://github.com/DavidAnson/markdownlint/blob/main/doc/md048.md) already covers this check, but does not include fix information, therefore can only be used to raise errors for manual fixing. This custom rule enforces the same style but includes fix information that can be used alongside our fix scripts.
+Markdown lint's [MD048 rule](https://github.com/DavidAnson/markdownlint/blob/main/doc/md048.md) already covers this check, but does not include fix information, therefore can only be used to raise errors for manual fixing. This custom rule enforces the same style but includes fix information that can be used alongside our fix scripts.
```

Comment:
No idea how this made it into the diff, given it's not showing any actual changes... Shouldn't need to do anything with it :shrug:

Reply:
Yeah I was confused sitting there trying to figure out why this file is in here lol